app/src/main/java/hous/release/android/util/component/ToDoUserScreen.kt

```diff
@@ -0,0 +1,124 @@
+package hous.release.android.util.component
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material.Divider
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import hous.release.android.R
+import hous.release.android.presentation.our_rules.type.ButtonState
+import hous.release.android.presentation.todo.viewmodel.UpdateToDoUiState
+import hous.release.android.util.theme.b2
+import kotlinx.coroutines.Job
+
+@Composable
+fun TodoUserScreen(
+    uiState: UpdateToDoUiState,
+    checkUser: (Int) -> Unit,
+    selectTodoDay: (Int, Int) -> Unit,
+    finish: () -> Boolean,
+    name: String,
+    putToDo: () -> Job,
+    hideKeyBoard: () -> Unit
+) {
+    val interactionSource = remember { MutableInteractionSource() }
+    Surface(
+        color = colorResource(id = R.color.white),
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(horizontal = 16.dp)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .clickable(
+                    interactionSource = interactionSource,
+                    indication = null
+                ) { hideKeyBoard() }
+        ) {
+            Text(
+                text = stringResource(id = R.string.to_do_manager),
+                style = b2,
+                color = colorResource(id = R.color.hous_black)
+            )
+            Spacer(modifier = Modifier.size(3.dp))
+            ToDoUserProfiles(
+                isEmptySelectedUser = uiState.selectedUsers.isEmpty(),
+                selectedUsers = uiState.selectedUsers
+            )
```

Comment:
```suggestion
            if(uiState.selectedUsers.isEmpty()){
                ToDoUserProfiles(selectedUsers = uiState.selectedUsers)            
            }
```
컴포넌트 안에 visibilty를 구분하는 flag를 넣는 것 보다 screen에 넣는 것이 screen 함수를 읽으면서
아 selectedUser가 비어 있으면 이 컴포넌트는 안 보이는 구나~
하며 이해하기 더 좋을 것 같다는 개인적인 생각입니다!

Reply:
오 그게 더 좋겠네요 배워갑니당~